README.md

```diff
@@ -82,6 +82,20 @@ See the [Reporter Interface section of the API documentation](https://github.com
 
 Looking for more examples? Check out the [examples folder](https://github.com/hapijs/good/tree/master/examples).
 
+## Reporter Type
```

Comment:
It doesn't seem that common though? Like I said I've only had one bug opened on "good" that was because of this issue.

Again, this is a documentation PR, so I'd like to keep the focus of the conversation on this pull request and not on new or additional features. We can discuss the check elsewhere if you want to open a new issue.